tasks/provision_service.rb

```diff
@@ -89,7 +89,7 @@ def provision(platform, inventory_location, vars)
     data = JSON.parse(vars.tr(';', ','))
     job_url = data['job_url']
   end
-  inventory_full_path = File.join(inventory_location, 'inventory.yaml')
+  inventory_full_path = File.join(Dir.pwd, '/spec/fixtures/litmus_inventory.yaml')
```

Comment:
Why is this one place not using `inventory_location` ? Would be useful to have a comment in the code explaining that.

Reply:
There is no need for this, setting it back to inventory_location. 
Thank you! :-)